packages/gatsby-transformer-remark/src/on-node-create.js

```diff
@@ -50,12 +60,16 @@ module.exports = async function onCreateNode(
 
     createNode(markdownNode)
     createParentChildLink({ parent: node, child: markdownNode })
+
+    return markdownNode
```

Comment:
Just to make easier to test--the functionality is already accomplished with createNode and createParentChildLink

Reply:
Is it worth mentioning this as a comment?

Reply:
@m-allanson I did mention it later (e.g. `// eslint`) but _probably_.

I'll add that before I merge!